pkg/kubelet/prober/prober_manager_test.go

```diff
@@ -294,6 +303,7 @@ func TestUpdatePodStatus(t *testing.T) {
 	m.readinessManager.Set(kubecontainer.ParseContainerID(probedReady.ContainerID), results.Success, &v1.Pod{})
 	m.readinessManager.Set(kubecontainer.ParseContainerID(probedUnready.ContainerID), results.Failure, &v1.Pod{})
 	m.startupManager.Set(kubecontainer.ParseContainerID(startedNoReadiness.ContainerID), results.Success, &v1.Pod{})
+	m.readinessManager.Set(kubecontainer.ParseContainerID(crashedNoReadiness.ContainerID), results.Success, &v1.Pod{})
```

Comment:
This case is called "crashed" but it is passing Success here - is that right?

Reply:
Yeah, see the thread here https://github.com/kubernetes/kubernetes/pull/96805#discussion_r530074501.

This is to test that a not started container with a previous ready status results in a not ready status. A comment explaining why this is `Success` here probably doesn't hurt.

Reply:
Yeah, to the uninitiated it seems wrong - tests have the same readability needs as real code :)

Reply:
a comment would help, but i wont block on it.